src/Development/IDE/Plugin/Completions/Logic.hs

```diff
@@ -236,11 +259,39 @@ mkPragmaCompl label insertText =
     Nothing Nothing Nothing Nothing Nothing (Just insertText) (Just Snippet)
     Nothing Nothing Nothing Nothing Nothing
 
+extendImportList :: String -> LImportDecl GhcPs -> Maybe [TextEdit]
+extendImportList name lDecl = let
+    f (Just range) ImportDecl {ideclHiding} = case ideclHiding of
+        Just (False, x) -> let
+            already_defined = Set.member name (Set.fromList [show y| y <- unLoc x])
+            start_pos = _end range
+            new_start_pos = start_pos {_character = _character start_pos - 1}
+            line = _line . _end $ range
+            char = (_character . _end $ range) + 3 + length name
+            end_pos = Position line char
+            new_range = Range new_start_pos end_pos
+            alpha = all isAlphaNum name
+            result = if alpha then ", " ++ name ++ ")"
+                else ", (" ++ name ++ "))"
+            in
```

Comment:
FIX: the `","` should only be inserted when the import list is not empty

Reply:
Ugh! Good catch. Thanks!